src/main/kotlin/com/refinedmods/refinedstorage/util/WorldUtils.kt

```diff
@@ -38,18 +40,21 @@ fun getFluidHandler(@Nullable tile: BlockEntity?, side: Direction?): IFluidHandl
         tile.getCapability(CapabilityFluidHandler.FLUID_HANDLER_CAPABILITY, side).orElse(null)
     } else null
 }
-
-fun getFakePlayer(world: ServerWorld, @Nullable owner: UUID?): FakePlayer {
-    if (owner != null) {
-        val profileCache: PlayerProfileCache = world.getServer().getPlayerProfileCache()
-        val profile: GameProfile = profileCache.getProfileByUUID(owner)
-        if (profile != null) {
-            return FakePlayerFactory.get(world, profile)
+*/
+    fun getFakePlayer(world: ServerWorld, owner: UUID?): PlayerEntity { // TODO: FakePlayer?
+        TODO("how to do fake players?")
+        /*
+        if (owner != null) {
+            val profileCache: PlayerProfileCache = world.getServer().getPlayerProfileCache()
+            val profile: GameProfile = profileCache.getProfileByUUID(owner)
+            if (profile != null) {
+                return FakePlayerFactory.get(world, profile)
+            }
         }
+        return FakePlayerFactory.getMinecraft(world)
+         */
```

Comment:
Can you undo the changes to this file so we don't get a conflict with my gui pr?